pkg/endpoint/bpf.go

```diff
@@ -366,41 +366,29 @@ func (e *Endpoint) addNewRedirects(m *policy.L4Policy, proxyWaitGroup *completio
 
 	desiredRedirects = make(map[string]bool)
 
-	// Ingress redirects.
-	if m != nil && m.HasRedirect() {
-		err, ff, rf = e.addNewRedirectsFromMap(true, desiredRedirects, proxyWaitGroup)
-		if err != nil {
-			return desiredRedirects, fmt.Errorf("unable to allocate ingress redirects: %s", err), nil, nil
+	for _, ingress := range []bool{true, false} {
+		dirLogStr := "ingress"
```

Comment:
really small nil, you could pass a map into the for loop

```golang
	for dirLogStr, ingress := range map[string]bool{"ingress": true, "egress": false} {
```

Reply:
Done!